README.md

```diff
@@ -42,3 +42,27 @@ python3.8 ./tools/experiment_manager.py run --dir ./experiments/muzero_random_ro
 python3.8 ./tools/experiment_manager.py play --file ./experiments/muzero_random_room_5x5/episodes/<EPISODE_FILENAME_HERE>
 ```
 
+
+## How to train an agent (with conda, without docker)
```

Comment:
Thanks for the contribution!

It would probably make more sense to split the description into two sections: "how to setup an environment" (with 2 subsections, for conda and docker), and "how to train an agent" so that stuff about creating and running an experiment is not duplicated.